src/wiseagents/core.py

```diff
@@ -1127,14 +1127,14 @@ def remove_context(cls, context_name: str):
             cls.contexts.pop(context_name)
     
     @classmethod
-    def clear_agents(cls):
+    def clear_agents_descriptions_dict(cls):
```

Comment:
Since this method is similar to `unregister_agent` but unregisters all the agents, just wondering if we should rename this method to `unregister_all_agents`?